### ISSUE TITLE
Complete the music player

### ISSUE THREAD

**Issue by ShalokShalom** [humhub/humhub#2180]
A user started to create a music player, which is near to complete now.
With some help can Humhub this essential new module: 

https://github.com/WebCrew/HumHub-Music-Player

**Comment by WebCrew** [comment]
Yeah thx to remember the guys, but I think You are false here. You should write that issue to the HH community, because the HumHub developers has nothing to do with community driven HH module/theme projects. They have enough work to do with working on HumHub it self. 

But it was a nice idea from You to try to get a bit more attention. @ShalokShalom

**Comment by ShalokShalom** [comment]
Ah i see. 

They consider the idea/project maybe as such beneficial, that they decide to make it an official HumHub module, who knows. :) 

And since developers are used to use Github, is that place here maybe suitable for such issues, since they wont hit the section in the HumHub Community so easy.

**Comment by Felli** [comment]
HumHub already has a music player and video player implemented.

**Comment by ShalokShalom** [comment]
Oh wow. Where? 

Is the community page still meant to be the demo?

**Comment by Felli** [comment]
You can test it on your community profile. 😄

**Comment by ShalokShalom** [comment]
Where?

![screenshot_20170808_101351](https://user-images.githubusercontent.com/6344099/29062534-5f01b7ce-7c22-11e7-86c4-9e48eb148de5.png)

**Comment by Felli** [comment]
It's core implemented, so once you upload a music or video file it'll render it.

**Comment by ShalokShalom** [comment]
Aha, ok. I want a music player, that runs in the background, while I use HubHub.

**Comment by Felli** [comment]
Hmm... Maybe change this issue to implement this feature? I can see it being possible.

**Comment by buddh4** [comment]
I think such a feature should not be part of the core, it's a good module idea, one problem I see is how to handle full page reloads, which are currently still used for example in forms/redirects etc. In my opinion the best option would be a module which creates a web radio stream which can also be opened by media players.

**Comment by ShalokShalom** [comment]
Sounds like the real issue is this type of page reloading?

**Comment by buddh4** [comment]
Yes, in some cases it would be rather complex to fully avoid such reloads, but you could just keep one additional tab open or in case of a a radio stream solution additinoally just open a stream file with a media player of your choice.
